### PR TITLE
Initial commit of pacman-reflector

### PR DESCRIPTION
**Pull request by dasJ**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+Arch Linux
+
+## Role Variables
+
+| Name                           | Default/Required | Description                                                                               |
+|--------------------------------|:----------------:|-------------------------------------------------------------------------------------------|
+| `reflector_connection_timeout` | `10`             | The number of seconds to wait before a connection times out                               |
+| `reflector_cache_timeout`      | `300`            | The cache timeout in seconds for the data retrieved from the Arch Linux Mirror Status API |
+| `reflector_sort`               | `score`          | Sort the mirrorlist                                                                       |
```
Comment (SlothOfAnarchy) [inline README.md]: you could mention other possibilities here

File: tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    src: reflector.sh.j2
+    dest: /usr/share/libalpm/scripts/reflector.sh
+    mode: 0755
+    owner: root
+    group: root
+  notify: Update mirrorlist
+
+- name: Create reflector hook
+  copy:
+    src: reflector.hook
```
Comment (SlothOfAnarchy) [inline tasks/main.yml]: Is it not necessary to prepend `files/`?

File: README.md (hunk 1 above)
Comment (dasJ) [inline README.md]: That is task of the man page. I don't want to copy this, because it might change in the future

File: tasks/main.yml (hunk 2 above)
Comment (dasJ) [inline tasks/main.yml]: It is not

**Pull request merged**